jenkins/catapult/runDockerBuildInnerBuild.py

```diff
@@ -19,8 +19,9 @@ def prepare(self, enable_code_coverage, out_dir):
 		self._prepare_directory(enable_code_coverage, out_dir)
 		self._prepare_environment_variables()
 
-		self.dispatch_subprocess(['ccache', '-M', '30G'])
-		self.dispatch_subprocess(['ccache', '-s'])
+		if not self.environment_manager.is_windows_platform():
+			self.dispatch_subprocess(['ccache', '-M', '30G'])
+			self.dispatch_subprocess(['ccache', '-s'])
```

Comment:
no ccache on win?

Reply:
ccache support on Windows is not great but even worst for MSVC :)
https://ccache.dev/platform-compiler-language-support.html